src/main/java/core/basesyntax/dto/FruitRecordDtoParseImpl.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.dto;
+
+import core.basesyntax.service.OperationType;
+import core.basesyntax.service.OperatorParse;
+import core.basesyntax.service.implementation.OperatorParseImpl;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FruitRecordDtoParseImpl implements FruitRecordDtoParse {
+    private static final String DELIMITER = ",";
+
+    @Override
+    public List<FruitRecordDto> parse(List<String> lines) {
+        List<FruitRecordDto> fruitRecordDtos = new ArrayList<>();
+        for (String line : lines) {
+            OperatorParse operatorParse = new OperatorParseImpl();
```

Comment:
make this variable belong to class instead of method, we don't need to create new instance per each method call